1-estrategia/canvas.md

```diff
@@ -0,0 +1,44 @@
+Unidad 1: Estrategia
+## **Tema: Lean UX canvas**<h1> 
+### Que es Lean UX canvas?<h2>
```

Comment:
Éste título, por tanto, sería de segundo nivel, en lugar de tercero:
```
## ¿Qué es Lean UX canvas?
```

Tendréis que revisar también los siguientes títulos.